lib/config_file.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+#config_file.py
+#
+#Combine standard login with option of OAuth2.0
+#
+import yaml
+
+def open_config():
+	CONFIG_FILE = "config.yml"
```

Comment:
no need to define this on a separate line btw since we only use it in the one place below

Reply:
updated